### ISSUE TITLE
VideoPlayer : cannot switch between MONO and STEREO movies

### ISSUE THREAD

**Issue by ogoguel** [googlevr/gvr-android-sdk#309]
When a Stereo movie is played after a Mono movie, the display remains in mono mode (instead of being displayed in Mono). Et vice et versa.

Sounds like the inputType (`TYPE_STEREO_OVER_UNDER` or `TYPE_MONO`) is only processed for the first movie being loaded.

A temporary workaround is to shutdown and recreate the  `VrVideoView` in the activity before loading/playing each new movie, 
```
   VrVideoView videoWidgetView = new VrVideoView(this);
   Options options = new Options();
   options.inputType = Options.TYPE_MONO;
   videoWidgetView loadVideo(uri, options);
   videoWidgetView.playVideo();
    ...
   videoWidgetView.pauseRendering();
   videoWidgetView.shutdown();
   videoWidgetView = null;
   videoWidgetView = new VrVideoView(this); 
   Options options = new Options();
    options.inputType = Options.TYPE_STEREO_OVER_UNDER;
   videoWidgetView loadVideo(uri, options);
   videoWidgetView.playVideo();
```
but even if it sort of works,  it seems that the previous views are still attached to the activity (you can see them when clicking on the close button (x))

Any help on that matter would be appreciated
Tested with SDK 1.10

**Comment by dav-cz** [comment]
Thanks for the bug report. We're able to reproduce, hopefully this should be fixed in our next version.

For your workaround, have you tried removing the VrVideoView from its parent layout before creating a new one, e.g. contentView.removeView(videoWidgetView);

**Comment by ogoguel** [comment]
Thanks for your feedback : looking forward for the next version.
I've finally fixed it by re-creating the whole activity every time I want to play a movie, but I'll have a look at removing the VrVideoView

**Comment by dav-cz** [comment]
This should be fixed in v1.20.0, thanks for your patience.
https://developers.google.com/vr/android/release-notes

**Comment by ogoguel** [comment]
Great. BTW why don't you open source the full sdk ? It would help the community helping you :)
Keep up the good work anyway.
